### PR TITLE
Added support for Bearer Token

### PR DESCRIPTION
**Pull request by rahul-aggarwal-avalara**
Added the method to pass Bearer Token

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -95,6 +95,18 @@ public function withLicenseKey($accountId, $licenseKey)
         return $this;
     }
 
+	/**
+     * Configure this client to use bearer token
+     *
+     * @param  string          $bearerToken     The private bearer token for your AvaTax account
+     * @return AvaTaxClient
+     */
+	public function withBearerToken($bearerToken)
```
Comment (nitinshirsat) [inline src/Client.php]: not sure if it is just github UI, but indentation looks off.

File: src/Client.php [hunk 2]
```diff
@@ -105,13 +117,24 @@ public function withLicenseKey($accountId, $licenseKey)
...
         // Set authentication on the parameters
-        if (!isset($guzzleParams['auth'])){
-            $guzzleParams['auth'] = $this->auth;
-        }
-        $guzzleParams['headers'] = [
-            'Accept' => 'application/json',
-            'X-Avalara-Client' => "{$this->appName}; {$this->appVersion}; PhpRestClient; 17.5.0-67; {$this->machineName}"
-        ];
+        if(count($this->auth) == 2)
+		{
```
Comment (nitinshirsat) [inline src/Client.php]: trivial : indentation is off.

**Comment by nitinshirsat** [comment]
Please fox the unit test https://github.com/avadev/AvaTax-REST-V2-PHP-SDK/blob/master/tests/basicWorkflowTest.php#L18
Details: 
https://travis-ci.org/avadev/AvaTax-REST-V2-PHP-SDK/builds/333142077?utm_source=github_status&utm_medium=notification

**Comment by rahul-aggarwal-avalara** [comment]
@nitinshirsat The unit test written is correct. This is failing because the environment variables are not set on Travis server, this is what we believe after reviewing the details provided here. The environment variables of username and password should be set from where the unit test is executed and then it should work properly. 
We are not making any further changes now, please let us know if anything is still required from our side.

**Pull request closed without merging**